src/UI/components/shared/footer.tsx

```diff
@@ -0,0 +1,31 @@
+import { useEffect } from 'react';
+import styles from '../../styles/Home.module.css';
+
+import { useAppSelector, useAppDispatch } from '../../state/hooks';
+import {
+  getApiVersion,
+  getUiVersion
+} from '../../state/configSlice';
+
+function Counter() {
```

Comment:
I'm not sure we want to confuse this component with collection stats - this should just be a Footer component.